scala/ppml/src/main/scala/com/intel/analytics/bigdl/ppml/FLClient.scala

```diff
@@ -37,6 +37,7 @@ class FLClient(val _args: Array[String]) extends GrpcClientBase(_args) {
   var psiStub: PSIStub = null
   var nnStub: NNStub = null
   var fgbostStub: FGBoostStub = null
+  privateKeyFilePath = "server.pem"
```

Comment:
var privateKeyFilePath = "server.pem"

Reply:
Parameter `privateKeyFilePath ` comes from it's father class.